config/initializers/constants.rb

```diff
@@ -0,0 +1,4 @@
+# frozen_string_literal: true
+
+SITE_SPLIT = /([a-z]{1,})(\W)(com)/.freeze
+MONEY_PREFIX = /^(R\W|\W)([0-9]{1,}\W[0-9]{1,})(\W[0-9]{1,})/.freeze
```

Comment:
Esses caras não devem estar dentro da classe que são utilizados? Não faz sentido deixar em um arquivo separado. Sem falar que é muito melhor usar as configurações customizadas do rails se for precisar separar e usar em vários lugares 🤔
https://guides.rubyonrails.org/configuring.html#custom-configuration

Reply:
novamente sugiro a leitura do livro clean code, eles se enaixam perfeitamente aqui, e não dentro de uma classe devido ao lixo que ele gera

Reply:
Certo, essa pratica é recomendada quando são constantes utilizadas com frequência por boa parte do código, não sendo o caso, e isso é uma prática do Rails 3 ou 4, sendo que essas constantes vão ser inicializadas por toda a aplicação quando subir e se manter la, veja que no Rails 5 e 6, no link que eu mandei, é possível configurar essas constantes no Rails configurations, o que é uma melhor prática.
https://stackoverflow.com/questions/4110866/ruby-on-rails-where-to-define-global-constants

Mas novamente, se não achar valido, basta resolver este comentário e seguir, leve em consideração que são apenas sugestões que eu considero melhorar o código, utilizando melhores práticas, seguindo o Ruby way, design patterns, removendo smells e afins.